app/models/candidate_interface/personal_details_form.rb

```diff
@@ -0,0 +1,19 @@
+module CandidateInterface
+  class PersonalDetailsForm
+    include ActiveModel::Model
+
+    attr_accessor :first_name, :last_name, :day, :month, :year, :nationality,
+                  :english_main
+
+    def name
+      "#{first_name} #{last_name}"
+    end
+
+    def main_language_english_options
```

Comment:
We wanted to use an `enum english_main: {}` here but couldn't figure out the exact mechanics of including `ActiveRecord::Enum` into this class... Feedback welcome on how to improve!

Reply:
I think this is alright!